ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -32,12 +32,14 @@
 {% include 'documentation/management-api-gnmi.j2' %}
 {## Management CVX #}
 {% include 'documentation/management-cvx.j2' %}
-{# management Console #}
+{## Management Console #}
 {% include 'documentation/management-console.j2' %}
 {## Management API HTTP #}
 {% include 'documentation/management-api-http.j2' %}
 {# Management API Models #}
 {% include 'documentation/management-api-models.j2' %}
+{## CVX #}
```

Comment:
The documentation template for CVX is still creating a level-1 heading.
"ip-http-client-source-interfaces" is part of the "Management" heading above, so please move CVX just below the http-client.